__tests__/utils/storage.spec.js

```diff
@@ -0,0 +1,23 @@
+const { Store } = require('data-store');
+const { getStoredValues, setStoreValues } = require('../../src/utils/storage');
+
+describe('storage', () => {
+  const getMock = jest.fn();
+  const setMock = jest.fn();
+  beforeAll(() => {
+    jest.spyOn(Store.prototype, 'get').mockImplementation(getMock);
+    jest.spyOn(Store.prototype, 'set').mockImplementation(setMock);
+  });
+  describe('getStoredValues', () => {
+    it('should retrieve stored values', () => {
+      getStoredValues('test');
+      expect(getMock).toHaveBeenCalled();
+    });
+  });
+  describe('setStoreValues', () => {
+    it('should save values', () => {
+      setStoreValues('test', 123);
+      expect(setMock).toHaveBeenCalled();
```

Comment:
maybe a toHaveBeenCalledWith

Reply:
I added this for the `set` value. I can't do this for `get` as its not called with anything, and any more mocking would be too deep into testing the package's functionality.